Dockerfile

```diff
@@ -19,8 +19,12 @@ FROM continuumio/miniconda3:4.7.12
 # Upgrade all packages to meet security criteria
 RUN apt-get update && apt-get upgrade -y && rm -rf /var/lib/apt/lists/*
 
-WORKDIR /workspace
-RUN mkdir assets
+
+RUN useradd --create-home max
+WORKDIR /home/max
+RUN cd /home/max && mkdir assets
```

Comment:
Do we need `cd /home/max`?

Reply:
Confirmed that it is not needed. Thanks for catching!

Reply:
Also, you might want to run `mkdir assets` as `max`, otherwise you don't know what permission issues you may run into

Reply:
```suggestion
RUN mkdir assets
```